src/commands/transfer_command/transfer_near_tokens_type/mod.rs

```diff
@@ -84,9 +84,37 @@ impl TransferNEARTokensAction {
         connection_config: Option<crate::common::ConnectionConfig>,
         sender_account_id: String,
     ) -> color_eyre::eyre::Result<Self> {
-        let amount: crate::common::NearBalance = match item.amount {
-            Some(cli_amount) => cli_amount,
-            None => TransferNEARTokensAction::input_amount(),
+        let amount: crate::common::NearBalance = match &connection_config {
+            Some(network_connection_config) => {
+                let account_balance: u128 = match crate::common::check_account_id(
+                    network_connection_config.clone(),
+                    sender_account_id.clone(),
+                )? {
+                    Some(account_view) => account_view.amount,
+                    None => 0,
+                };
```

Comment:
Convert `account_balance` to `NearBalance` once, and then use it instead of passing `u128` around.

```suggestion
                let account_balance: crate::common::NearBalance = match crate::common::check_account_id(
                    network_connection_config.clone(),
                    sender_account_id.clone(),
                )? {
                    Some(account_view) => crate::common::NearBalance::from_yoctonear(account_view.amount),
                    None => crate::common::NearBalance::from_yoctonear(0),
                };
```